AFQ/afq_bids_entities.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "bidsafq",
+    "entities": [
+      {
+        "name": "odfmodel",
+        "pattern": "[_/\\\\]+odfmodel-([a-zA-Z0-9]+)"
+      },
+      {
+        "name": "coordsys",
+        "pattern": "[_/\\\\]+coordsys-([a-zA-Z0-9]+)"
+      },
+      {
+        "name": "trkmodel",
+        "pattern": "[_/\\\\]+trkmodel-([a-zA-Z0-9]+)"
```

This entity takes the values `prob` or `det`? I am not sure that the word `model` is the most accurate for this. This was referred to as "method" in [this comment on bep16](https://github.com/bids-standard/bids-bep016/issues/43#issue-1069019873), so this would become:

```suggestion
        "pattern": "[_/\\\\]+trkmethod-([a-zA-Z0-9]+)"
```

WDYT?